plutus-core/untyped-plutus-core/src/NewUntypedPlutusCore.hs

```diff
@@ -0,0 +1,24 @@
+-- | Will replace UntypedPlutusCore.
```

Comment:
The main purpose is to avoid the circular dependency problem because the current `UntypedPlutusCore` has to import `UntypedPlutusCore.Parser`. And also cleaning things up a bit.